SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -205,6 +205,19 @@
     </PropertyGroup>
   </Target>
 
+  <Target Name="CleanProjectFolders" AfterTargets="Clean" Condition=" $(SonarQubeTempPath) != '' ">
+    <ItemGroup>
+      <ProjectSpecificOutFiles Include="$(SonarQubeOutputPath)\$(MSBuildProjectName)_*\*.*"/>
```

Comment:
What I meant in my first comment is that it seems like it's not possible to use wildcard with directories, for example:
`<ProjectSpecificOutDirs Include="$(SonarQubeOutputPath)\$(MSBuildProjectName)_*"/>`